BoldGrid/boldgrid-backup
# bgbkup-cli

Wiki: https://github.com/BoldGrid/boldgrid-backup/wiki/Site-Check-and-Restore-CLI

Fixes #12 -- Site Check and logging.

# Testing Instructions
1. [x] Perform a full backup.
1. [x] Change directory to: `wp-content/plugins/boldgrid-backup/`.
1. [x] Run the command: `php cli/bgbkup-cli.php check log log_level=LOG_DEBUG`.
1. [x] Break a PHP file, such as `wp-config.php`.
1. [x] Run the check command again.
1. [x] Review the log file: `bgbkup-cli.log`.

Email testing: Test in an environment that has outgoing email capabilities.
# Testing Instructions
1. [ ] Perform a full backup.
1. [ ] Change directory to: `wp-content/plugins/boldgrid-backup/`.
1. [ ] Run the command: `php cli/bgbkup-cli.php check log log_level=LOG_DEBUG notify email=you@domain.com`, using your email address.
1. [ ] Break a PHP file, such as `wp-config.php`.
1. [ ] Run the check command again.
1. [ ] Review the log file: `bgbkup-cli.log`.
1. [ ] Check for a notification email message.
